collector/collector.go

```diff
@@ -74,27 +74,14 @@ func (c *Collector) Start() error {
 			Description: "Lambda Collector",
 			Version:  Version,
 		},
-		ParserProvider: c.parserProvider,
+		ConfigMapProvider: c.mapProvider,
 		Factories:      c.factories,
 	}
 	var err error
 	c.svc, err = service.New(params)
 	if err != nil {
 		return err
 	}
```

Comment:
`Start()` should now take a context and invoke `go c.svc.Run(ctx)`.

Reply:
Done.